src/components/pages/career-details/new-career-form/career-form-fields.tsx

```diff
@@ -0,0 +1,156 @@
+import React from 'react';
+import { ErrorCode, FileError } from 'react-dropzone';
+import { Trans, useTranslation } from 'react-i18next';
+import styled from 'styled-components';
+import { SectionHeader } from '../../../content/section-header/section-header';
+import { Checkbox } from '../../../forms/checkbox/checkbox';
+import { FileDropper } from '../../../forms/file-dropper/file-dropper';
+import { StyledErrorMessage } from '../../../forms/text-input/text-input';
+import { ContentBlockContainer } from '../../../layout/content-block-container';
+import { TextStyles } from '../../../typography';
+import { Button } from '../../../ui/buttons/button';
+import { StyledLink } from '../../contact/contact-form';
+
+const PRIVACY_POLICY = 'https://satellytes.jobs.personio.de/privacy-policy';
+const MAX_SIZE = 20 * 1024 * 1024;
+
+const StyledButton = styled(Button)`
+  margin-top: 48px;
+`;
+
+const TextWrapper = styled.div`
+  margin: 24px 0;
+
+  ${TextStyles.textR}
+`;
+
+export const CareerDetailsSubmitButton = ({ isSubmitting, errors }) => {
+  const { t } = useTranslation();
+  return (
+    <Button type="submit" disabled={isSubmitting}>
+      {!errors?.api
+        ? t<string>('career.action.send')
+        : t<string>('career.action.again')}
+    </Button>
+  );
+};
+
+export const MandatoryFieldText = () => {
+  return (
+    <TextWrapper>
+      <Trans i18nKey={'career.mandatory-field'} />
+    </TextWrapper>
+  );
+};
+
+export const CareerDetailsFileText = () => {
+  return (
+    <TextWrapper>
+      <Trans i18nKey={'career.info-text'} />
+    </TextWrapper>
+  );
+};
+
+export const CareerDetailsCheckbox = ({ control }) => {
+  const { t } = useTranslation();
+
+  return (
+    <Checkbox
+      name="privacy"
+      label={
+        <Trans i18nKey={'career.privacy-policy'}>
+          <span>
+            Hiermit bestätige ich, dass ich die
+            <StyledLink to={PRIVACY_POLICY}>Datenschutzerklärung</StyledLink>
+            zur Kenntnis genommen habe
+          </span>
+        </Trans>
+      }
+      control={control}
+      rules={{ required: t<string>('career.error.approval') }}
+    />
+  );
+};
+
+export const CareerDetailsFileUpload = ({
```

Comment:
much better grouped like that instead of defining it inline in the form render part, or how does it fell to you ?

Reply:
I am ambiguous about this.
It is still a large component in career-form, since some of the props have still to be passed. 
But it is shorter and that is helpful